amgut/templates/secondary_survey.html

```diff
@@ -0,0 +1,84 @@
+{% extends sitebase.html %}
+{% from wtforms import validators %}
+{% block head %}
+<import >
+<script type="text/javascript">
+    $(document).ready(function() {
+      $("div[id^=supp_]").hide();
+    });
+</script>
+<style>
+input[type="checkbox"] {
```

Comment:
was specific to the animal page that I modeled it off of, but now covers 2 pages.